spec/fixtures/example.csv

```diff
@@ -0,0 +1,8 @@
+,SIMPSON,HOMER DUFF,19/01/1980 00:00,,N
+,MOUSE,MICKEY,05/02/1980 00:00,,N
+,MUFFET,LITTLE MISS,06/02/2000 00:00,,N
```

Comment:
Sorry, I'm using your PR to think out loud 😂 - if the CSV ever had Mr/Miss etc in it, we might need to think about how we match on records. The risk is we return "No record found" when there is a matching record.

Reply:
Yeah that's a good point, I think we should strip the names of any titles, I was thinking that we need to standardise the case for the values so I'll add to this.

Reply:
@steventux nice! could defs be in a follow-up PR if you wanted.